.travis.yml

```diff
@@ -1,7 +1,8 @@
 language: python
+dist: xenial
```

Comment:
Is that necessary?

Reply:
This was also added in the branch that I submitted recently. It is a good practice since we don't know when travis-ci will change their default. Their default (14.04 LTS) is now out of support and thus we shouldn't expect package updates to continue. My thoughts on stuff like this is to always be on the latest LTS (and then if possible test one back as well) that is available (18.04 however I don't think travis-ci has support for it yet).